### PR TITLE
fix: [ANDROAPP-6457] limit max height of expanded cards in maps

### PR DESCRIPTION
**Pull request by Balcan**
## Description
Please include a summary of the change and include the related jira issue if it exists.

[ jira issue ](https://dhis2.atlassian.net/browse/ANDROAPP-

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] 11.X - 13.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

**Review by xavimolloy — APPROVED** [review]

File: app/src/main/java/org/dhis2/usescases/teiDashboard/ui/mapper/TeiDashboardCardMapper.kt [hunk 1]
```diff
@@ -265,4 +265,5 @@ class TeiDashboardCardMapper(
         this.filter { it.first.valueType() != ValueType.IMAGE }
             .filter { it.first.valueType() != ValueType.COORDINATE }
             .filter { it.first.valueType() != ValueType.FILE_RESOURCE }
+            .filter { it.second.value()?.isNotEmpty() == true }
```
Comment (mmmateos) [inline app/src/main/java/org/dhis2/usescases/teiDashboard/ui/mapper/TeiDashboardCardMapper.kt:268]: Is this fix related with the issue?

Comment (xavimolloy) [reply app/src/main/java/org/dhis2/usescases/teiDashboard/ui/mapper/TeiDashboardCardMapper.kt:268]: It is not with the issue itself, but Marcos mentioned it in the troubleshooting channel that we have apparently lost this functionality and as it's an easy fix I thought I would include it
